### ISSUE TITLE
Error when upgrading to 0.9.0

### ISSUE THREAD

**Issue by satazor** [postcss/postcss-loader#67]
```
ERROR in ./~/css-loader?importLoaders=1&sourceMap!./~/postcss-loader!./src/routes/search/components/ResultsListItem.css
Module build failed: TypeError: Cannot read property 'postcss' of undefined
    at Processor.normalize (/Users/satazor/Work/npms/npms-www/node_modules/postcss-loader/node_modules/postcss/lib/processor.js:53:18)
    at new Processor (/Users/satazor/Work/npms/npms-www/node_modules/postcss-loader/node_modules/postcss/lib/processor.js:23:29)
    at postcss (/Users/satazor/Work/npms/npms-www/node_modules/postcss-loader/node_modules/postcss/lib/postcss.js:55:12)
    at Object.module.exports (/Users/satazor/Work/npms/npms-www/node_modules/postcss-loader/index.js:57:5)
 @ ./src/routes/search/components/ResultsListItem.css 4:14-172 13:2-17:4 14:20-178
```

```css
.page-search .results-list-item {
    position: relative;
    margin-bottom: 30px;
    padding: 0 0 0 50px;
}
```

**Comment by ai** [comment]
@satazor show your webpack config

**Comment by ai** [comment]
Fixed in 0.9.1
